ClientSecurity/Configuring_IPS_Notifications.md

```diff
@@ -0,0 +1,61 @@
+# Configuring Intrusion Prevention System (IPS) Notifications
```

Comment:
Tried to add this in with filled out fields, the HTML code still displaying in preview mode...